cmd/mv.go

```diff
@@ -39,13 +40,28 @@ func mv(cmd *cobra.Command, args []string) error {
 	var mvErrors []error
 	var relocationArgs []*files.RelocationArg
 
+	re := regexp.MustCompile("[^/]+$")
 	for _, argument := range argsToMove {
-		arg, err := makeRelocationArg(argument, destination+"/"+argument)
-		if err != nil {
-			relocationError := fmt.Errorf("Error validating move for %s to %s: %v", argument, destination, err)
-			mvErrors = append(mvErrors, relocationError)
+
+		argumentFile := re.FindString(argument)
+		lastCharDest := destination[len(destination)-1:]
+
+		if lastCharDest == "/" {
+			arg, err := makeRelocationArg(argument, destination + argumentFile)
+			if err != nil {
```

Comment:
Can you split this out of the if-else block? Lines 51-56 are exactly the same as lines 59-64.

Reply:
ok! I have updated the cmd/mv.go file with requested changes.